sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/CommunicationIdentityClientBuilder.java

```diff
@@ -139,6 +141,18 @@ public CommunicationIdentityClientBuilder addPolicy(HttpPipelinePolicy customPol
         return this;
     }
 
+        /**
+     * Sets the client options for all the requests made through the client.
+     *
+     * @param clientOptions {@link ClientOptions}.
+     * @return The updated {@link CommunicationIdentityClientBuilder} object.
+     * @throws NullPointerException If {@code clientOptions} is {@code null}.
+     */
+    public CommunicationIdentityClientBuilder clientOptions(ClientOptions clientOptions) {
+        this.clientOptions = Objects.requireNonNull(clientOptions, "'clientOptions' cannot be null.");
```

Comment:
The  `clientOptions` have applicationId which should get into `UserAgent` here.
https://github.com/Azure/azure-sdk-for-java/blob/ea3f1d485aee21df0118c2a3ad5b66b45c538d4b/sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/CommunicationIdentityClientBuilder.java#L272

This is an old PR shows how UserAgent should be populated with `applicationId`

https://github.com/Azure/azure-sdk-for-java/pull/16428/files#diff-57d5c797a106560d37148554ddcceb69f85c68f35b2a257823fbcc69b24a28aaR171

Reply:
Thanks for this. Addressed